aerospike_helpers/expressions/map.py

```diff
@@ -799,6 +871,8 @@ def __init__(self, ctx: "TypeCTX", return_type: int, begin: "TypeKey", end: "Typ
             bin if isinstance(bin, _BaseExpr) else MapBin(bin),
         )
         self._fixed = {_Keys.RETURN_TYPE_KEY: return_type}
+        if inverted:
+            self._fixed[_Keys.RETURN_TYPE_KEY] |= ReturnType.LIST_RETURN_INVERTED
```

Comment:
```suggestion
            self._fixed[_Keys.RETURN_TYPE_KEY] |= ReturnType.MAP_RETURN_INVERTED
```

I think this should use the map constant? This occurs in multiple places, please check this file and change them all.

Reply:
Fixed